sdk/core/amqp-common/changelog.md

```diff
@@ -1,8 +1,8 @@
-
 ### 2019-7-3 1.0.0-preview.6
 
 - Include bug fix where token audience was being set to work only with Event Hubs. The appropriate token audience now needs to be explicitly set on the credentials at time of creation. [PR 4098](https://github.com/Azure/azure-sdk-for-js/pull/4098)
 - Document usage of AAD tokens and known issue with token expiry usage on MSI based credentials with AAD authentication. [PR 4148](https://github.com/Azure/azure-sdk-for-js/pull/4148)
+- Since browser doesn't differentiate between the various kinds of service communication errors that occur when using WebSockets, a utility has been added to identify and classify such errors. [PR 2936](https://github.com/Azure/azure-sdk-for-js/pull/2936)
```

Comment:
Name the utility that was added, saves the user the time taken to search for it :)